cmd/ctr/commands/run/run_windows.go

```diff
@@ -55,6 +55,10 @@ func NewContainer(ctx gocontext.Context, client *containerd.Client, context *cli
 		id = context.Args().First()
 		opts = append(opts, oci.WithSpecFromFile(context.String("config")))
 		cOpts = append(cOpts, containerd.WithContainerLabels(commands.LabelArgs(context.StringSlice("label"))))
+		if context.Bool("cni") {
```

Comment:
Please check L135. IIUC, `run_windows.go` should align with `run_unix.go`.

Reply:
updated, thanks